### DISCUSSION TITLE
400 response during signin

### DISCUSSION THREAD

**Discussion by MGW-SumathyV** [nocodb/nocodb#3623] [General]
Hi,

While signin I am getting below error. It was working fine earlier.

{"stack":"Error: select * from `nc_users_v2` where `email` = 'venu.sumathy@gmail.com' limit 1 - Can't add new command when connection is in closed state\n    at Connection._addCommandClosedState (/ServerSide/packages/nocodb/node_modules/mysql2/lib/connection.js:148:17)\n    at Connection.query (/ServerSide/packages/nocodb/node_modules/mysql2/lib/connection.js:546:17)\n    at /ServerSide/packages/nocodb/node_modules/knex/lib/dialects/mysql/index.js:134:18\n    at new Promise (<anonymous>)\n    at Client_MySQL2._query (/ServerSide/packages/nocodb/node_modules/knex/lib/dialects/mysql/index.js:128:12)\n    at Client_MySQL2.query (/ServerSide/packages/nocodb/node_modules/knex/lib/client.js:168:17)\n    at Runner.query (/ServerSide/packages/nocodb/node_modules/knex/lib/runner.js:151:36)\n    at /ServerSide/packages/nocodb/node_modules/knex/lib/runner.js:40:23\n    at /ServerSide/packages/nocodb/node_modules/knex/lib/runner.js:277:24\n    at runMicrotasks (<anonymous>)","message":"select * from `nc_users_v2` where `email` = 'venu.sumathy@gmail.com' limit 1 - Can't add new command when connection is in closed state","fatal":true}

could you please let me know how to resolve this?

Thanks and Regards,
Sumathy Venugopal

**Comment by wingkwong** [comment]
Can you share the project info and what was your previous version that was working fine?

**Reply by MGW-SumathyV** [reply]
> Can you share the project info and what was your previous version that was working fine?

This is the only version we are using and it was working fine earlier and now having signin issues

nocodb  "version": "0.91.10",

nc-gui   "version": "0.11.35"

 Where can we get the project info ?

**Reply by wingkwong** [reply]
Are you using Docker? It doesn't work after upgrading to the latest version right?

Project Info can be retrieved here.

![image](https://user-images.githubusercontent.com/35857179/190074620-ed7ac375-0fdc-4b78-8a04-88b02e46b8e2.png)

**Reply by MGW-SumathyV** [reply]
Yes we are using docker. But we haven't upgraded. We are using the same version right from beginning.
We are getting this error while signing in.We have created several projects. Which project info should I send

**Reply by o1lab** [reply]
@MGW-SumathyV : how are you sure about nc-gui and nocodb version if its within a docker ? Reason I ask, we do not provide the nc-gui version if its on docker in project info ?

**Reply by MGW-SumathyV** [reply]
As I was not well I couldn't respond immediately. Sorry for the delayed response.
We have referred to package.json and we provided the version numbers. Please find below the project info which we copied from UI

Node: **v14.20.0**
Arch: **x64**
Platform: **linux**
Docker: **false**
Database: **mysql2**
ProjectOnRootDB: **true**
RootDB: **mysql2**
PackageVersion: **0.91.10**
